mozmeao/birdbox
# Change AI Builder's Challenge contact form destination

## Description

This changeset changes the Basket newsletter ID that the Builder's Challenge form uses, and also changes the internal naming of the form to be more broad (and not just AI focused)

- [X] I have manually tested this.

### Issue

No issue filed - Slack request

### Testing

* Add an Innovations Contact Form to an Innovations Content Page and select the Builder's Challenge option (note the new name)
* Submit the form using a throwaway yourhandle+something@mozilla.com email address 
* Confirm you get signed up to the right newsletter by accepting the subscription and then viewing your preferences
